packages/esm-patient-registration-app/translations/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "addRelationshipButtonText": "Add Relationship",
+  "address1": "",
```

Comment:
Shouldn't these be coming from the backend, rather than be set to empty strings?

Reply:
And [one day the backend to do that will exist](https://issues.openmrs.org/browse/RESTWS-892).